arena.py

```diff
@@ -326,20 +328,35 @@ def buy_xp_round(self) -> None:
         if arena_functions.get_gold() >= 4:
             mk_functions.buy_xp()
 
+
+    def load_aguments(self):
+        augments = self.comps_manager.CURRENT_COMP()[2]
+        return augments
```

Comment:
Returning an immediate inline variable
```suggestion
    def load_aguments(self):
        return self.comps_manager.CURRENT_COMP()[2]
```